content/blog/2018-03-13-weathercan.md

```diff
@@ -0,0 +1,500 @@
+---
+slug: "weathercan"
+title: Integrating data from weathercan
+package_version: 0.2.5
+authors:
+  - name: Steffi LaZerte
+    url: http://steffilazerte.ca
+date: 2018-03-02
+categories: blog
+topicid: 000
+tags:
+  - r
+  - community
+  - software
+  - review
+  - package
+  - data-access
+  - weather
+  - Canada
+  - meteorology
+  - tutorial
+output:
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+I love working with R and have been sharing the love with my friends and colleagues for almost seven years now. I'm one of those really annoying people whose response to most analysis-related questions is "You can do that in R! Five minutes, tops!" or "Three lines of code, I swear!" The problem was that I invariably spent an hour or more showing people how to get the data, load the data, clean the data, transform the data, and join the data, before we could even start the "five minute analysis". With the advent of [`tidyverse`](https://www.tidyverse.org) data manipulation has gotten much, much easier, but I still find that data manipulation is where most new users get stuck.
+
+This is one of the reasons why, when I designed [`weathercan`](http://github.com/ropensci/weathercan), I tried as hard as possible to make it simple and straightforward. `weathercan` is an R package designed to make it easy to access historical weather data from [Environment and Climate Change Canada (ECCC)](http://climate.weather.gc.ca/historical_data/search_historic_data_e.html). It downloads, combines, cleans, and transforms the data from multiple stations and across long time frames. So when you access ECCC data, you get everything in one dataset. Nifty, eh?
+
+However, there is a certain point at which the user has to take matters into their own hands. Although downloading data with `weathercan` is fairly straight forward, weather data often needs to be integrated into other data sets. Depending on the other data this can be a tricky step. You may want to combine `weathercan` data with other types of measurements (e.g., river water samples on a specific day), or summarize and join it with data on other scales (e.g. temporal or spatial).
```

Comment:
A couple of sentences are similar enough in tone that you could omit one of them or revise to condense. 
- "Depending on the other data this can be a tricky step. "
- "Integrating weathercan data can be straightforward, but often it’s a bit convoluted."

Reply:
Maybe you could combine and condense the paragraphs that start "However, there is a certain point ..." and "Integrating weathercan data can be straightforward,..."